assets/js/base/components/product-order-select/style.scss

```diff
@@ -0,0 +1,3 @@
+p.wc-block-order-select {
```

Comment:
`p` required here?

Reply:
There was a specificity issue, but I refactored it to use a `div` instead of a `p` so we don't have this issue: 67c9c5ccdd68e0e867a04936d266d15c7af6ff29.